addons/mail/static/src/messaging/component/composer_text_input/composer_text_input.js

```diff
@@ -143,10 +195,79 @@ class ComposerTextInput extends Component {
         this._textareaRef.el.style.height = (this._textareaRef.el.scrollHeight) + "px";
     }
 
+    /**
+     * Validates a keyword in order to trigger mentions search.
+     *
+     * @private
+     * @param delimiter
+     * @param beginningOnly
+     * @return {*}
+     */
+    _validateKeyword(delimiter, beginningOnly) {
```

Comment:
what does it mean "validate keyword"? Docstring doesn't really help understanding neither.